website/content/api-docs/system/remount.mdx

```diff
@@ -6,16 +6,32 @@ description: >-
   endpoint.
 ---
 
-# `/sys/remount`
+# Remount ( API )
 
-The `/sys/remount` endpoint is used remount a mounted backend to a new endpoint.
+The Remount documentation details the endpoints required to trigger and monitor the status of a remount operation.
 
 ## Move Backend
 
-This endpoint moves an already-mounted backend to a new mount point. 
+The `/sys/remount` endpoint moves an already-mounted backend to a new mount point. This works for both secret engines
+and auth methods.
+
+It can be used to move the backend across namespaces as well, with the namespaces supplied as prefixes
```

Comment:
If the implied reference is the endpoint, I recommend stating that at the beginning of the sentence and replace "It".

Reply:
[29ce383](https://github.com/hashicorp/vault/pull/14314/commits/29ce3830382cc5a0df24a043911868d4bf0323eb)